src/main/sh/mommy

```diff
@@ -358,6 +359,54 @@ done
 
 shift "$((OPTIND - 1))"
 
+# do the symlinking
+if [ -n "$opt_rename" ]; then
+    if [ $# -lt 1 ]; then
+        die "mommy expected a file path to rename"
+    fi
+    echo "mommy is locating her shell script"
+    findoldpath=0
+    oldpath="$(whereis mommy | sed 's/^mommy://')"
```

Comment:
i think this is not a suitable method for locating `mommy`. sure, you may find `mommy`, but how do you know that's the instance you want to replace? what if someone has multiple installations of `mommy`? say, one in `/usr/bin` and one in `~/.local/bin`?

instead of trying to find "the" mommy binary, you should find the _current_ mommy binary, and link to that